doc/en/developer/source/policies/review.rst

```diff
@@ -0,0 +1,76 @@
+.. _review:
+
+Review
+======
+
+Reviewing code is an invaluable process that is used to ensure that the highest quality code possible gets committed into GeoServer at all times. 
```

Comment:
I'd go for "acceptable" quality code. Highest is far away from what we have and what we demand.